packages/genesys-spark-components/src/components/beta/gux-time-zone-picker/gux-time-zone-picker.tsx

```diff
@@ -20,14 +20,20 @@ export class GuxTimeZonePickerBeta {
   @Element()
   private root: HTMLElement;
 
+  @Prop({ mutable: true })
+  value: string;
+
+  @Prop()
+  workspaceDefault: string;
```

Comment:
Could you add some annotation to indicate the difference between workspace default and custom default?